pkg/tso/keyspace_group_manager.go

```diff
@@ -915,10 +941,12 @@ func (kgm *KeyspaceGroupManager) deleteKeyspaceGroup(groupID uint32) {
 		am.close()
 		kgm.ams[groupID] = nil
 	}
+
+	kgm.deletedGroups[groupID] = struct{}{}
```

Comment:
The previous group cannot be cleaned, right?

Reply:
Yes, it relies on the deletion event currently.